### PR TITLE
Coding standard

### PR DESCRIPTION
**Pull request by pamil**
Based on #217.

### CODE REVIEW

**Comment by lchrusciel** [comment]
fail :(

**Comment by pamil** [comment]
@lchrusciel not anymore!

File: tests/Application/app/autoload.php [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <?php
 
-use Doctrine\Common\Annotations\AnnotationRegistry;
+declare(strict_types=1);
```
Comment (lchrusciel) [inline tests/Application/app/autoload.php]: missing blank line

**Comment by lchrusciel** [comment]
👍

**Pull request merged**